CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
+    - Branch from last tag on `develop`
+
+### Pull requests
+
+- As soon as you have made 1 commit in a feature branch, put up a *draft* pull request
+- Keep pull requests small :skull: 
+- :warning: No pre-emptive reviews on PR drafts, **unless** the PR author @-mentions with this specific request
+- When you think you're done, mark PR ready for review, see below (Merge Process)
+
+## Merge Process (into `develop`)
+
+- Create PR from `feature/...` against `develop` (PR author)
+- Describe work in initial comment (PR author)
+    - Reference any related issues (use, e.g., `Fixes #n` or `- Related: #n`)
+    - What does the new code do?
+    - Optional: What should reviewers look at specifically
+    - Information on how to review:
+        - E.g.
+          ```bash
+          pip install ./
+          pytest test/
+          ```
+- Request review (PR author)
+    - Eligible reviewers:
+        - Python code: @led02, @sdruskat, @jkelling
+        - Documentation:
+            - Workflow: @all
+            - Project: @all
```

Comment:
**TODO:**

1. Add missing review targets
2. Add your handles to targets

Reply:
IDK if I'm eligible to be included here. I'm simply not a good Python coder...